solidity/contracts/dao/FlexibleVotingVault.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.25;
+
+import {IVault} from "symbiotic-core/src/interfaces/vault/IVault.sol";
+
+import {FlexVotingClient} from "flexible-voting/src/FlexVotingClient.sol";
+
+contract FlexibleVotingVault is FlexVotingClient {
+    IVault public immutable vault;
+
+    constructor(address _vault, address _governor) FlexVotingClient(_governor) {
+        vault = IVault(_vault);
+    }
+
+    function _rawBalanceOf(
+        address _user
+    ) internal view override returns (uint256) {
+        return vault.activeBalanceOf(_user);
+    }
+
+    // betrays the client interface by asking for timestamp instead of block number
```

Comment:
I think you'll need to expand the docs here if you gonna do this

Reply:
Did you talk with ScopeLift about this?

Reply:
I think this should be fine since the function is overriden and not using the underlying history struct which requires a block number